TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -397,24 +414,31 @@ void runReplayWithIOStreams(Duration observedPacketConnectionTimeout,
         }
     }
 
+    ConcurrentHashMap<UniqueRequestKey, Boolean> liveRequests = new ConcurrentHashMap<>();
+
     private BiConsumer<UniqueRequestKey, HttpMessageAndTimestamp>
     getRecordedRequestReconstructCompleteHandler(ReplayEngine replayEngine) {
         return (requestKey, request) -> {
             replayEngine.setFirstTimestamp(request.getFirstPacketTimestamp());
 
+            liveTrafficStreamLimiter.addWork(1);
             var requestPushFuture = transformAndSendRequest(replayEngine, request, requestKey);
             requestFutureMap.put(requestKey, requestPushFuture);
-            requestPushFuture.map(f->f.whenComplete((v,t)-> log.atTrace()
-                    .setMessage(()->"Summary response value for " + requestKey + " returned=" + v).log()),
+            liveRequests.put(requestKey, true);
+            requestPushFuture.map(f->f.whenComplete((v,t)->{
+                        liveRequests.remove(requestKey);
+                liveTrafficStreamLimiter.doneProcessing(1);
```

Comment:
nit: Indentation looks off here

Reply:
I'm likely going to wait on this until I get checkstyle going on the whole repo.